code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -59,6 +59,14 @@
 	var/timer = 450
 
 /obj/machinery/doomsday_device/process()
+	if(z != ZLEVEL_STATION)
+		minor_announce("DOOMSDAY DEVICE OUT OF RANGE OF STATION, ABORTING", "ERROR ER0RR $R0RRO$!R41.%%!!(%$^^__+ @#F0E4", 1)
+		SSshuttle.emergencyNoEscape = 0

```

Comment:
the doomsday device turning on stops the shuttle from escaping, then it turns off and lets it escape again
the only actual effect is resetting the shuttle timer